### PR TITLE
[WIP] Improvement/add msu full sum

### PR DESCRIPTION
**Pull request by blag001**

### CODE REVIEW

File: src/ogkush.js [hunk 1]
```diff
@@ -12566,12 +12566,55 @@ class OGInfinity {
         toFormatedNumber(Math.floor(dSumP + dSumM + this.json.flying.deuterium), null, true)
       )
     );
+
+    let fullSumresMsu = createDOM("div", { class: "ogl-res ogl-sum-symbol" });
+
+    let tradeRate = this.json.options.tradeRate;
```
Comment (baiiko) [inline src/ogkush.js:12572]: You can use const instead of let for object.

File: src/ogkush.js [hunk 2]
```diff
@@ -12566,12 +12566,55 @@ class OGInfinity {
...
+        },
+        toFormatedNumber(Math.floor(fullSumMsu), null, true)
+      )
+    );
+
+    let sum = createDOM("div", { class: "smallplanet smaller ogl-summary" });
     sum.appendChild(createDOM("div", { class: "ogl-sum-symbol" }, "ΣΣ"));
     sum.appendChild(sumres);
+
+
```
Comment (baiiko) [inline src/ogkush.js:12596]: remove one line ? i dont know if coding style is very strict

File: src/ogkush.js [hunk 3]
```diff
@@ -12035,12 +12037,54 @@ class OGInfinity {
...
+    );
+
     sum.appendChild(createDOM("div", { class: "ogl-sum-symbol" }, "ΣΣ"));
     sum.appendChild(sumres);
+
+    let sumMsuSideDiv = sum.appendChild(createDOM("div", { class: "ogl-sum-symbol tooltip", "title": standardUnit.unitType(true)}, standardUnit.unitType()));
+    sum.appendChild(sumresStandardUnit);
+
     list.appendChild(sum);
+
```
Comment (guideloince) [inline src/ogkush.js:12067]: drop one blank line

File: src/ogkush.js [hunk 4]
```diff
@@ -12035,12 +12037,54 @@ class OGInfinity {
...
+    let sumMSU = createDOM("div", { class: "smallplanet smaller ogl-summary" });
+    sumMSU.appendChild(createDOM("div", { class: "ogl-sum-symbol tooltip", "title": standardUnit.unitType(true)}, "ΣΣ "+standardUnit.unitType()));
+    sumMSU.appendChild(sumresStandardUnit.cloneNode(true));
+    list.appendChild(sumMSU);
+
     if (document.querySelectorAll(".moonlink").length == 0) {
       divMoonSum.style.display = "none";
       moonSumSymbol.style.display = "none";
+
```
Comment (guideloince) [inline src/ogkush.js:12077]: here too

**Review by Bishop341-B — APPROVED** [review]

**Pull request merged**